### PR TITLE
Add Boston/Cambridge spots

### PR DESCRIPTION
**Pull request by FreddieV4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -407,6 +409,30 @@ Wifi | Power | Coffee | Address | Open Hours
...
+
+> Open workspace with whiteboard walls, great coffee, moderately busy.
+
+Webpage: http://www.districthallboston.org/
+
+Wifi | Power | Coffee | Address | Open Hours
+---- | ----- | ------ | ------- | ----------
+open | ✔ | ✔ | [175 Northern Ave, Boston, MA 02210](https://goo.gl/maps/odxUyRJT2bS2) | Mon-Fri 08:00-02:00, Sat 12:00-02:00, Sun 12:00-21:00
+
+## Boston, MA <a id="cambridge-ma"></a>
```
Comment (diasdavid) [inline README.md]: Seems like you wanted to write "Cambrige" here instead of Boston

**Comment by diasdavid** [comment]
thank you @FreddieV4, made a note, seems like you might have wanted to type "Cambrige, MA" instead of "Boston, MA" in the second time it appears, could you confirm please?

**Comment by FreddieV4** [comment]
Oops. I meant to write Cambridge. I'll just add another commit to fix it?

**Comment by FreddieV4** [comment]
@diasdavid Fixed it!

**Comment by diasdavid** [comment]
Thank you @FreddieV4 :)

**Pull request merged**